File: Python-MySQL - Without Docker/app/files/models/report_type.py
from files import db


class ReportType(db.Model):
    __tablename__ = 'report_type'
    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    description = db.Column(db.String(50), nullable=False)

    def __repr__(self):
        return f"{{\"id\": \"{self.id}\", \"description\": \"{self.description}\"}}"

File: Node-MySQL/app/models/ReportType.js
'use strict';
const {
    Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
    class ReportType extends Model {
        /**
         * Helper method for defining associations.
         * This method is not a part of Sequelize lifecycle.
         * The `models/index` file will call this method automatically.
         */
        static associate(models) {
            // define association here
        }
    }
    ReportType.init({
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        description: {
            type: DataTypes.STRING(50),
            allowNull: false,
            validate: {
                notNull: { msg: "Description required!" }
            }
        }
    }, {
        sequelize,
        modelName: 'ReportType',
        tableName: 'ReportType',
        timestamps: false
    });
    return ReportType;
};